### PR TITLE
feat: undrop tabulars

### PR DESCRIPTION
**Pull request by twuebi**

### CODE REVIEW

File: src/pages/warehouse/[id].namespace.[nsid].vue [hunk 1]
```diff
@@ -165,7 +165,13 @@
                     >
                   </td>
                 </template>
-
+                <template v-slot:item.actions="{ item }">
+                    <v-icon
+                      color="error"
+                      :disabled="!myAccess.includes('delete')"
```
Comment (twuebi) [inline src/pages/warehouse/[id].namespace.[nsid].vue:171]: We got undrop on table and undrop all on namespace. If we want to offer the undrop button here and enable users who do not have undrop all on namespace to undrop, we'll have to send `getTableAccessById` for each table row which doesn't sound very efficient 

Dropping tabulars from this namespace view currently checks for delete on namespace access and does not additionally check via `getTableAccessById` although there is also a can_drop on table / view.

@v-kessler @c-thiel

**Review by c-thiel — COMMENTED** [review]
We need to improve the message if a tabular already exists - its in the type.
Should we show the "type" ore improve the "message"?

image

**Comment by c-thiel** [comment]
In the deleted tab, we should also show the "delted_at" timestamp

**Comment by c-thiel** [comment]
In the top right corner I have two user icons (see my comment in https://github.com/lakekeeper/console/pull/44#pullrequestreview-2491203380). This wasn't introduced here, was it?

File: src/pages/warehouse/[id].namespace.[nsid].vue (hunk 1 above)
Comment (twuebi) [reply src/pages/warehouse/[id].namespace.[nsid].vue:171]: discussed with @c-thiel, we're gonna remove disabling from drop & undrop and just show an error if we fail to drop & undrop

**Review by v-kessler — DISMISSED** [review]

**Comment by twuebi** [comment]
added error type to the snackbar @c-thiel

File: src/pages/warehouse/[id].namespace.[nsid].vue [hunk 2]
```diff
@@ -165,7 +165,13 @@
                     >
                   </td>
                 </template>
-
+                <template v-slot:item.actions="{ item }">
```
[the remark this answers is not in the document]
Comment (twuebi) [reply src/pages/warehouse/[id].namespace.[nsid].vue]: created #56 as a follow-up issue instead

**Review by v-kessler — APPROVED** [review]

**Pull request merged**